src/js/tracker.js

```diff
@@ -916,6 +923,23 @@
 		}
 
 		/**
+		 * Check that *both* optimizely and optimizely.data exist
+		 *
+		 * @param property optimizely data property
+		 * @param snd optional nested property
+		 */
+		function getOptimizelyXData(property, snd) {
+			var data;
+			if (windowAlias.optimizely && windowAlias.optimizely.data) {
+				data = windowAlias.optimizely.get(property);
+				if (typeof snd !== 'undefined' && data !== undefined) {
+					data = data[snd]
+				}
+			}
+			return data
+		}
```

Comment:
Alright, I'll give this a final test on a testbed setup with Optimizely X tomorrow morning.